cmd/demo/main.go

```diff
@@ -14,7 +14,7 @@ func main() {
 	// Depending on how big of a mailbox you give your listeners,
 	// and how busy your publishers are, this number becomes very
 	// important to avoid bleeding messages through Err()
-	p := spub.New(time.Millisecond * 1)
+	p := spub.New(time.Second * 10)
```

Comment:
I purposefully have errors in the demo to demonstrate how to handle them and to ensure that all errored messages are accounted for, so i'm removing it to demonstrate what happens in a vacuum